modules/ROOT/pages/whats-new-in-studio.adoc

```diff
@@ -1,3 +1,27 @@
 = What’s New in Studio
 
 Anypoint Studio 7.15.0 improves the developer experience by enhancing Studio performance for a number of users, strengthens the asset import experience, and adds more to CloudHub deployment options.
+
+== Improved Performance for Customers on Windows
+
+Windows users now experience improved performance in Studio, as they can use the built-in tool to exclude Studio from Windows Defender scans.
+
+== Enhanced Experience Importing API Specifications or Fragments
+
+Studio enhances importing experience of API specifications or fragments from Design Center improving the ability to search and provide more details displaying:
```

Comment:
I'm not sure what this sentence is saying. Maybe split into two?

Reply:
Thanks for your comment!
I think the new version is easier to read and understand.